specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml

```diff
@@ -17,22 +13,22 @@ options:
     emitter-output-dir: "{project-root}/.."
     examples-directory: "./examples"
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/widgets.json"
-  "@azure-tools/typespec-python":
-    emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
+  "@azure-tools/typespec-python":    
+    package-dir: "azure-contoso-widgetmanager"
     package-mode: "dataplane"
-    package-name: "azure-contoso-widgetmanager"
+    package-name: "{package-dir}"    
   "@azure-tools/typespec-csharp":
+    package-dir: "Azure.Contoso.WidgetManager"
     clear-output-folder: true
-    emitter-output-dir: "{csharp-sdk-folder}/sdk/{service-directory-name}/{namespace}/src"
     model-namespace: false
-    namespace: Azure.Contoso.WidgetManager
+    namespace: "{package-dir}"    
```

Comment:
@m-nash can we default the namespace to package-dir to eliminate this needed in the config? Also can we eliminate some of the other options with nices defaults?

Reply:
https://github.com/Azure/autorest.csharp/issues/3408